wgpu-native/src/lib.rs

```diff
@@ -184,7 +184,7 @@ pub type AdapterId = Id;
 type AdapterHandle = hal::Adapter<B>;
 pub type DeviceId = Id;
 type DeviceHandle = Device<B>;
-pub type QueueId = Id;
+pub type QueueId = DeviceId;
```

Comment:
Type aliases for for `QueueId` and `CommandEncoderId` aren't clear to me, why aren't they just `Id`?

Reply:
just to signify that they are going to be used in the relevant storages, i.e. the code will access the device storage with QueueId and the command buffer storage with CommandEncoderId